core/core.go

```diff
@@ -101,47 +101,47 @@ type Config struct {
 	P2PWebSocketsPort int `envvar:"P2P_WEBSOCKETS_PORT" default:"60559"`
 	// EthereumRPCURL is the URL of an Etheruem node which supports the JSON RPC
 	// API.
-	EthereumRPCURL string `envvar:"ETHEREUM_RPC_URL" json:"-"`
+	EthereumRPCURL string `envvar:"ETHEREUM_RPC_URL" json:"ethereumRPCURL"`
```

Comment:
this exports this env var by default, should probably comb through if we log it in any place explicitly